packages/transaction-controller/src/TransactionController.test.ts

```diff
@@ -2469,7 +2523,7 @@ describe('TransactionController', () => {
         to: ACCOUNT_2_MOCK,
         hash: externalTransactionHash,
         id: externalTransactionId,
-        chainId: toHex(5),
+        chainId: toHex(1), // note that this doesn't match the globally selected chain Id
```

Comment:
are you adding these as a way of testing that the transaction chainId doesn't need to match? If so we should either move it into a separate test and specifically call out that we're testing that.

Reply:
reverted this change here https://github.com/MetaMask/core/pull/3664/commits/b56e66af4ce4f659a1e9abf5b29278886a5eb5a6

will address this again by splitting it out into a separate scenario in another branch with other updates to the tests